src/platform/forms-system/src/js/web-component-patterns/titlePattern.js

```diff
@@ -94,7 +94,12 @@ export const titleUI = (titleOption, descriptionOption) => {
           </legend>
         )
       ) : (
-        <Title title={title} description={description} headerLevel={3} />
+        <Title
+          title={title}
+          description={description}
+          headerLevel={headerLevel}
+          classNames={classNames}
```

Comment:
This looks like a good catch. Were `classNames` omitted on purpose before? 🤔

Reply:
I don't believe so, this was work that Robert and I collaborated on and went back and forth on so those attribute assignments got lost in the shuffle.